### ISSUE TITLE
Multisig on testnet "Create Transaction" remains greyed out

### ISSUE THREAD

**Issue by abradshaw** [sparrowwallet/sparrow#612]
When trying to send from a multisig wallet on testnet, the Create Transaction button remains greyed out

**Platform**: Linux (RPM)
**Version**: 1.65
**Network**: testnet

**Reproducer**
Create a multisig wallet (I did 1 sig using software and 2 using 2 different Ledgers)
Send some money from a faucet into the multisig wallet
Once its arrived to to sent it to a new testnet address

**Results**
Even though the Pay To, Label and Amount fields are filled in, the Create Transaction remains unclickable (grey)

![Screenshot from 2022-07-02 12-02-42](https://user-images.githubusercontent.com/404108/176995924-7ca47f2a-e746-49c5-bded-80c1ecfb50f1.png)

**Comment by abradshaw** [comment]
I have rerun this process in debug mode and when entering the amount I see the following error in the terminal

ERROR Exception in thread "JavaFX Application Thread"
com.sparrowwallet.drongo.protocol.ProtocolException: P2WSH scriptPubKey hash does not match constructed witness script hash

Full message attached

[sparrow-multisig-error.txt](https://github.com/sparrowwallet/sparrow/files/9032794/sparrow-multisig-error.txt)

**Comment by craigraw** [comment]
It is possible to recreate this with software keystores? If so please provide the steps below. I'd like to try recreate this error to understand it better - I have obviously many testnet multisig wallets that work without problems, so I'm trying to identify what might be different in your setup.

**Comment by abradshaw** [comment]
I have tried 3 other multisig setups (1 all software 2 or 3, one 2 of 2 with both hardware etc) and they all seem to work fine, its just this first one that was a 2 or 3 with 1 x software and 2 x ledger. So yeah I can't seem to reproduce. Its no big deal as its testnet but it does concern me.

**Comment by abradshaw** [comment]
By the way I love Sparrow - its easily the best wallet - thanks for making a Linux version

**Comment by craigraw** [comment]
Thanks! 

It’s a bit of a mystery - given it’s testnet, any chance I could get the wallet output descriptor to look into it? Can send privately to craig@sparrowwallet.com if so.

**Comment by abradshaw** [comment]
I will email it to you, thanks

**Comment by craigraw** [comment]
Ok, no problem. To get the output descriptor, go to the Settings tab and click the 'Edit...' button on the right next to the descriptor field. The descriptor will appear in the dialog that pops up, and is a few lines of text in the textbox containing the xpubs etc.

**Comment by abradshaw** [comment]
Hi

Ok I'm back now, here is the info you requested. If you need any info like
output from debug logging, just let me know

wsh(sortedmulti(2,[8c36e86f/48h/1h/0h/2h]tpubDFS7WkCDCNPLdmq8Xqe32RAigwGaRuXJyGWcuQ1LQdRen2aEc1pCq2EoNXyCZ8Xh2ch9yUe6uJ79ViYZbwTV2XnhSem83UhTTrbPfBa66hh/<0;1>/*,[24160894/48h/1h/0h/2h]tpubDEyQ9Zo33C3HSC4aqZdVVfaqVy3jknGLqj115tcVqAewxNohZYVe7rWD1FXSmPdLiee1LqUz8233Xy63joCGovfTCm3Mxuaazq2Vu8jmRuu/<0;1>/*))#uwq868je

Thanks, Adrian

**Comment by abradshaw** [comment]
Just to close this off - Craig and I were able to work out what went wrong, via email. I was trying to setup a 2 of 3 multisig with 2 different ledger HW devices. Both were plugged in at the same time. What I did wrong was when I scanned in the 2nd and 3rd keystores from the 2 ledgers, I must have selected the same ledger device for both keystores. So 2 of the 3 keystores were identical. 

Craig mentioned that he will add protection from this in a future version - awesome.

Since then I have created multiple multisig wallets and all work perfectly. I think the tip for working with 2 ledgers is to only have one plugged in at a time, then there can be no mistakes
